LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,394 @@
+to avail **Priority Support** as set forth in this Exhibit B. 
+
+1.  **Scope**
+
+    Support includes problem fixing, including but not limited
+    to correction of faults and errors (bugs), in official stable releases
+    of the Licensed Software.
+
+    Support will be provided for official stable release of the Licensed Software 
+    for a duration of two (2) years from the date of release.
+
+    Support for an older official stable release is not guaranteed but may be provided 
+    on a best effort basis.
+
+2. **Validity of Support Hours**
+
+    Unused Support Hours expire 12 months from date of purchase.
+    
+3.  **Ownership**
+
+    SixtyFPS may choose to include all changes, including problem fixes
+    and enhancements, to the Licensed Software and make it available for
+    use to all users.
+
+4.  **Problem Isolation**
+
+    Customer will perform initial problem isolation before a problem is
+    reported to SixtyFPS. Customer is advised to recreate potential
+    problems using the most recent release when reporting problems to
+    SixtyFPS. It is recommended that Customer briefly test each new
+    release of the Licensed Software before distribution of any Customer's Products incorporating such
+    release.
+
+5.  **Reporting a problem**
+
+    Customer shall use issue tracking systems as provided by SixtyFPS to
+    report problems. The problem description shall be in English.
+
+6.  **Problem recreation at SixtyFPS**
+
+    If SixtyFPS cannot reasonably replicate the setup under which the problem can
+    be recreated, then in SixtyFPS’s sole discretion, SixtyFPS may ask Customer to provide
+    the necessary setup or elect to recreate the problem at the Customer location. Any costs incurred
+    will be invoiced to the Customer.
+
+7.  **Errors in Modified Source**
+
+    SixtyFPS is not responsible for errors introduced by Customer
+    modifications to Licensed Software. Defects must be demonstrated on
+    an unmodified version of Licensed Software provided by SixtyFPS.
+    
+8. **Non Licensed Software related problems**
+
+    SixtyFPS will make reasonable efforts to provide workarounds for problems 
+    that are not directly related to the Licensed Software but are present in 
+    the Customer's Products built with the Licensed Software. Customer understands 
+    that workarounds may not be possible for all problems.
+
+9. **Operating Systems Updates**
+
+    SixtyFPS shall consider making commercially reasonable efforts to provide 
+    Customers with updates to the Licensed Software for newly released versions 
+    of operating systems. If, however, extensive changes to the Licensed Software 
+    are required, SixtyFPS may charge additional fees for the changes. Customer is 
+    under no obligation to accept such updates that require additional fees and 
+    SixtyFPS is not obligated to provide any specific operating system updates.
+
```

Comment:
Would it make sense to move the "Exhibit" sections up to where the headline were before? I would find that clearer, but then IANAL:-)